entries/w/wp.pl.json

```diff
@@ -0,0 +1,14 @@
+{
+  "WP": {
+    "domain": "wp.pl",
+    "url": "https://www.wp.pl/",
```

Comment:
As `https://wp.pl/` already redirects to `https://www.wp.pl/` this line is unnecessary.
```suggestion
```